src/test/ui/consts/const-eval/simd/insert_extract-fail.stderr

```diff
@@ -0,0 +1,16 @@
+error: any use of this value will cause an error
+  --> $DIR/insert_extract-fail.rs:14:14
+   |
+LL |     unsafe { simd_insert(x, 0_u32, 42_i16) }.0
+   |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   |              |
+   |              Inserting `i16` with size `2` to a vector element place of size `1`
+   |              inside call to `foo` at $DIR/insert_extract-fail.rs:19:19
```

Comment:
I don't know if this error message is clear or not. IIRC the intrinsics do not allow type mismatches, e.g., even though `i8` and `u8` have the same size, you can't insert an `i8` into a `u8x2` vector. 

IIRC that does lead to a monomorphization time error inside `rustc_codegen_llvm`, but this is not something that `typeck` checks, yet. (I think, otherwise this would have failed to type check)

Reply:
As long as there's no typeck error it's fine doing this here